xml_struct/src/tests.rs

```diff
@@ -173,3 +176,67 @@ fn int_as_element() {
         "Serializing `u64` should result in bare text content"
     );
 }
+
+#[test]
+fn bool() {
```

Comment:
suggestion: for both coverage and consistency, there maybe should be a `bool_as_element` test similar to `int_as_element`.